web/bulk_rendering/build.sh

```diff
@@ -0,0 +1,54 @@
+#! /bin/bash
+# 
+# Compile the KeymanWeb Recorder module for use with developing/running engine tests.
```

Comment:
bulk_rendering module

Reply:
Copy-paste, my arch-nemesis... we meet again.